components/dashboard/notification-list.tsx

```
@@ -0,0 +1,261 @@
+      case "newest":
+        return (
+          new Date(b._creationTime).getTime() -
+          new Date(a._creationTime).getTime()
+        );
+      case "oldest":
+        return (
+          new Date(a._creationTime).getTime() -
+          new Date(b._creationTime).getTime()
+        );
+      case "priority":
+        const priorityOrder = { high: 3, medium: 2, low: 1 };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)